builder/azure/arm/step_publish_to_shared_image_gallery.go

```diff
@@ -72,58 +86,60 @@ func getSigDestination(state multistep.StateBag) SharedImageGalleryDestination {
 	}
 }
 
-func (s *StepPublishToSharedImageGallery) publishToSig(ctx context.Context, subscriptionID string, sourceID string, sharedImageGallery SharedImageGalleryDestination, miSGImageVersionEndOfLifeDate string, miSGImageVersionExcludeFromLatest bool, miSigReplicaCount int64, location string, diskEncryptionSetId string, tags map[string]string) (string, error) {
-	replicationRegions := make([]galleryimageversions.TargetRegion, len(sharedImageGallery.SigDestinationReplicationRegions))
-	for i, v := range sharedImageGallery.SigDestinationReplicationRegions {
+func (s *StepPublishToSharedImageGallery) publishToSig(publishArgs PublishArgs) (string, error) {
```

Comment:
nit: The argument name can just be args since we know the type. `publishArgs PublishArgs` is a bit repetitive. 
```suggestion
func (s *StepPublishToSharedImageGallery) publishToSig(publishArgs PublishArgs) (string, error) {
```

Reply:
Done in newest commit